src/hooks/screen/useChatScreenHook.ts

```diff
@@ -33,7 +33,7 @@ interface ScrollContextProps {
 }
 export const ScrollContext = React.createContext<ScrollContextProps | null>(null);
 
-function useChatScreenHook(type: 'SIGNED' | 'ANONYMOUS'): UseChatScreenHook {
+function useChatScreenHook(type?: 'SIGNED' | 'ANONYMOUS'): UseChatScreenHook {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The type parameter for the `useChatScreenHook` function has been made optional. This could potentially lead to issues if the rest of your code assumes that `type` will always be either 'SIGNED' or 'ANONYMOUS'. If there are cases where `type` can be undefined, you should add appropriate checks in the body of the function to handle this case.
```diff
- function useChatScreenHook(type?: 'SIGNED' | 'ANONYMOUS'): UseChatScreenHook {
+ function useChatScreenHook(type: 'SIGNED' | 'ANONYMOUS' = 'SIGNED'): UseChatScreenHook {
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->